app/ts/components/AbbreviatedValue.tsx

```diff
@@ -0,0 +1,16 @@
+import { parseNumericTerm } from '../library/utilities.js'
+
+export const AbbreviatedValue = ({ floatValue }: { floatValue: number }) => {
+	const { coefficient, exponent } = parseNumericTerm(floatValue)
+
+	if (exponent < 0) {
+		const leadingZeros = Math.abs(exponent) - 1
+		return <>0.<small>{'0'.repeat(leadingZeros)}</small>{coefficient}</>
```

Comment:
if the input value (floatValue), is negative, this will show it as a positive number. In Interceptor we need to display negative values at times.

Reply:
Created a separate issue here for negative numbers support #220